spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -2996,6 +2996,33 @@ include::{snippets}/task-logs-documentation/get-logs-by-task-id/curl-request.ado
 
 include::{snippets}/task-logs-documentation/get-logs-by-task-id/http-response.adoc[]
 
+[[api-guide-resources-task-logs-by-task-executionid]]
+==== Get the task execution log by task execution id
+
+To retrieve the logs of the task execution by the task execution id, query the `/tasks/logs/taskexecutionid/<TaskExecutionId>` endpoint by using the HTTP `GET` method.
```

Comment:
This endpoint is for the same resource ("/tasks/logs") as `/tasks/logs/<ExternalTaskExecutionId>` with the only difference being the type of id is instead `/tasks/logs/<InternalTaskExecutionId>`. I wonder if `/tasks/logs/<task-id>?id-type=(internal|external)` where by default type = 'external' and then for this new api would look like `/tasks/logs/<task-id>?type=internal`.

Just thinking out loud here. If there is already precedence for this pattern in our API then this approach wins via consistency.

Reply:
👍